pandas/core/arrays/base.py

```diff
@@ -385,7 +387,7 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
-    def __contains__(self, item) -> bool | np.bool_:
+    def __contains__(self, item: Any) -> bool | np.bool_:
```

Comment:
I think we type this with Label elsewhere?

Reply:
@jreback I did a grep and only saw `Any`

Reply:
`Label` is for index values, this is an array, so `Any` is correct I think

Reply:
Label was an alias for Union[Hashable, None] to workaround a mypy bug that has since been fixed. Now mypy recognizes None as being Hashable.

wrt Any

from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#incomplete-stubs

> Included functions and methods must list all arguments, but the arguments can be left unannotated. Do not use Any to mark unannotated arguments or return values.

from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions

> When adding type hints, avoid using the Any type when possible. Reserve the use of Any for when:

> the correct type cannot be expressed in the current type system; and
> to avoid Union returns (see above).

> Note that Any is not the correct type to use if you want to indicate that some function can accept literally anything: in those cases use object instead.

Reply:
Based on that explanation, it should be `object` here

Reply:
I've used `object`, but had to do a `type ignore` when `any()` was called on a comparison.  I couldn't figure out how to work around that.  See latest commit.